forte/data/data_store.py

```diff
@@ -1228,6 +1224,87 @@ def add_generics_raw(
         entry = self._new_generics(type_name, tid)
         return self._add_entry_raw(Generics, type_name, entry)
 
+    def add_multipack_generic_raw(
+        self, type_name: str, tid: Optional[int] = None
+    ) -> Tuple[int, int]:
+        r"""This function adds a multi pack generic entry with ``type_name`` to
+        the current data store object. Returns the ``tid`` and the ``index_id``
+        for the inserted entry in the list. This ``index_id`` is the index
+        of the entry in the ``type_name`` list.
+
+        Args:
+            type_name: The fully qualified type name of the new Generics.
+            tid: ``tid`` of multi pack generic entry.
+
+        Returns:
+            ``tid`` of the entry and its index in the (``type_id``)th list.
+
+        """
+        tid: int = self._new_tid() if tid is None else tid
+        entry = [None, None, tid, type_name]
+        entry += self._default_attributes_for_type(type_name)
+        return self._add_entry_raw(MultiPackGeneric, type_name, entry)
+
+    def add_multipack_link_raw(
```

Comment:
Feels like we have a bit too many `add_raw` functions, maybe we should somehow merge them. We can put that to another issue.